desktop/ui/src/main/java/org/datacleaner/windows/ElasticSearchDatastoreDialog.java

```diff
@@ -96,20 +211,53 @@ protected void onChange(DocumentEvent event) {
                 validateAndUpdate();
             }
         });
+        _usernameTextField.getDocument().addDocumentListener(new DCDocumentListener() {
+            @Override
+            protected void onChange(DocumentEvent event) {
+                validateAndUpdate();
+            }
+        });
+        _passwordField.getDocument().addDocumentListener(new DCDocumentListener() {
```

Comment:
Looks like the same DCDocumentListener is created 3 times in this block of code. Would be easier to reuse